Makefile

```diff
@@ -212,7 +212,7 @@ clean_meta:
 #   make clean_generated
 .PHONY: clean_generated
 clean_generated:
-	find . -type f -name $(GENERATED_FILE_PREFIX)\* | xargs rm -f
+	find . -type f -name $(GENERATED_FILE_PREFIX)\* | grep -v "[.]/staging/.*" | xargs rm -f
```

Comment:
Why should we not remove files in staging folder? Will be good to add a comment here explaining that